coda/coda_mdstore/models.py

```diff
@@ -70,6 +70,10 @@ class Node(models.Model):
     This model defines a storage node for the mdstore
     """
 
+    STATUS_CHOICES = [
+        ('1', 'Active'),
+        ('2', 'Inactive'),
```

Comment:
Since we are using digits here, anyone else think it would make sense to use '0' for inactive an '1' for active?

Reply:
The thought _did_ occur to me. That or even using just a boolean field: https://docs.djangoproject.com/en/2.2/ref/models/fields/#booleanfield. But I also don't think it's an issue the way it currently is, so I'm fine either way

Reply:
I thought of using Boolean Field initially, but then I thought we may want to see 'active/inactive' as status in html pages rather than having a boolean value. I don't mind changing the choices to use '0' and '1'.

Reply:
You got the 0 and 1, but we need to swap which one means active. The intent was for 0 to be _inactive_ and 1 to be _active_.